app/domain/logs.rb

```diff
@@ -351,42 +351,42 @@ module AuthnJwt
       )
 
       AddingJwtClaimToValidate = ::Util::TrackableLogMessageClass.new(
-        msg: "Adding JWT claim '{0-claim-name}' to list of mandatory claims to be validated",
+        msg: "Adding JWT claim, '{0-claim-name}' to list of mandatory claims to be validated...",
         code: "CONJ00069D"
       )
 
       CheckingJwtClaimToValidate = ::Util::TrackableLogMessageClass.new(
-        msg: "Checking if JWT claim '{0-claim-name}' is mandatory to validate",
+        msg: "Checking if JWT claimת '{0-claim-name}' is mandatory to validate...",
```

Comment:
you're in Hebrew :) and notice there are 2 ,
Checking if JWT claim, '{0-claim-name}', must be validated...

Reply:
Goot catch thanks